CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Next release
 
+* [CHANGE] Changed ThanosRuler custom resource field alertmanagersURL type from string to []string` (#3067)
```

Comment:
Didn't know we started writing next release as part of features? Do we have it document somewhere?

Reply:
We don't have any official practice around this AFAIK, I just wanted to make sure it was included in the next changelog since it is a breaking change.